### DISCUSSION TITLE
Bitbucket. ssh: handshake failed: knownhosts: key mismatch

### DISCUSSION THREAD

**Discussion by ghost** [argoproj/argo-workflows#11175] [General]
Is there a way to update known hosts file for argo workflows? I am running on k8s and all of the sudden started getting "ssh: handshake failed: knownhosts: key mismatch". I am runnig version v3.4.7. I suspect bitbucket rotated the keys ...

**Comment by terrytangyuan** [comment]
v3.4.8 should include the fix

**Reply by ghost** [reply]
Thank you. is there a way to manually update them without bumping up the version?

**Reply by terrytangyuan** [reply]
You can mount a new ssh known hosts file to /etc/ssh/
